src/string_id.h

```diff
@@ -3,6 +3,8 @@
 
 #include <string>
 #include <type_traits>
+#include <memory>
+#include "debug.h"
```

Comment:
Well spotted. Thanks. I had added them for debugging purposes, but then forgot to delete.